tracker_doctor_test.go

```
@@ -325,3 +326,316 @@ func TestPinnedDippinVersionMatchesGoMod(t *testing.T) {
+
+// TestDoctor_GitRequires_BareRepoReportsError pins the PR #235 Copilot fix:
+// a bare repo passes `git rev-parse --git-dir` but has no work tree, so
+// `git commit` / `git merge` (the operations `requires: git` workflows
+// actually use) would fail. Doctor must report Error (or Warn under warn
+// policy), not OK, so the user gets the actionable remediation before
+// running the workflow.
+func TestDoctor_GitRequires_BareRepoReportsError(t *testing.T) {
+	requireGit(t)
+	tmp := t.TempDir()
+	bare := filepath.Join(tmp, "bare.git")
+	cmd := exec.Command("git", "init", "--bare", "-q", bare)
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)